src/org/labkey/test/tests/AdvancedImportOptionsTest.java

```diff
@@ -76,7 +76,7 @@
     @Override
     protected String getProjectName()
     {
-        return "AdvancedImportOptions";
+        return null;
```

Comment:
interesting, this deviates from the usual pattern- maybe helpful to leave a comment/note why?